api/conf/conf.yaml

```diff
@@ -17,23 +17,25 @@
 
 conf:
   listen:
-    host: 127.0.0.1          # `manager api` listening ip or host name
-    port: 8080               # `manager api` listening port
+    host: 127.0.0.1 # `manager api` listening ip or host name
```

Comment:
I prefer the old style, it is clearer and easier to read.

Reply:
The newer is formatted by tools 🤔 then it seems that it's not a good idea to have too many comments in this file.